packages/networks/src.ts/index.ts

```diff
@@ -179,6 +205,23 @@ const networks: { [name: string]: Network } = {
 
     bnb: { chainId: 56, name: "bnb" },
     bnbt: { chainId: 97, name: "bnbt" },
+
+    // hedera networks
+    hederaMainnet: {
```

Comment:
lets remove the `testnet` one and change the behaviour of the default provider to:

- `hethers.providers.getDefaultProvider('mainnet' | 'testnet' | 'previewnet' | 290, 291, 292)`